packages/thirdweb/src/utils/nft/parseNft.ts

```diff
@@ -28,6 +28,7 @@ export type NFTMetadata = {
   background_color?: string;
   properties?: Record<string, unknown>;
   attributes?: Record<string, unknown>;
+  image_url?: string;
```

Comment:
when do we need this? isnt it always `image` ?

Reply:
Jonas believes sometimes they can be image_url